dndtroops-persistence/src/main/java/cz/muni/fi/pa165/dndtroops/entities/Administrator.java

```diff
@@ -1,7 +1,47 @@
 package cz.muni.fi.pa165.dndtroops.entities;
 
+/**
+ * Created by Miroslav Macor
+ */
 public class Administrator {
+    private String name;
+    private long id;
 
-    // TODO
 
+    public Administrator(String name, long id) {
+        this.name = name;
+        this.id = id;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public void setName(String name) {
+        this.name = name;
+    }
+
+    public long getId() {
+        return id;
+    }
+
+    public void setId(long id) {
+        this.id = id;
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
+        if (o == null || getClass() != o.getClass()) return false;
```

Comment:
Do not use `getClass()` use `instanceof`. With `getClass()` you are forbidding inheritance. (at least that is what has been told me).